apps/bot/commands/slash/get-answer-count.ts

```diff
@@ -0,0 +1,64 @@
+import { SlashCommandBuilder } from 'discord.js'
+import { SlashCommand } from '../types.js'
+import {
+  getCorrectAnswersCount,
+  isUserProfilePublic,
+} from '../../db/actions/users.js'
+
+export const command: SlashCommand = {
+  data: new SlashCommandBuilder()
+    .setName('get-answer-count')
+    .setDescription('Get the number of your accepted answers')
+    .addUserOption((user) =>
+      user
+        .setName('user')
+        .setDescription('The user to get number of marked answers for')
+        .setRequired(false),
+    )
+    .setDMPermission(false),
+
+  async execute(interaction) {
+    // Get the user option
+    const userOption = interaction.options.getUser('user')
+
+    // Use the ID of the user option if it's provided, otherwise use the ID of the interaction user
+    const userId = userOption?.id || interaction.user.id
+    const userArgProvided = !!userOption
+    const count = await getCorrectAnswersCount(userId)
+    if (!count) {
```

Comment:
Maybe it's worth it to add a comment explaining that the response should be the same for when user is not in db and when user has no recorded answers. Right now at first glance I would question whether this code differentiates `null` and `0` the same way accidentally or intentionally.

Reply:
So, I guess it would be better to consider that count is 0, if it doesn't exist.